modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,9 +35,24 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes both user-visible new features and internal enhancements and fixes. The most significant new features and capabilities include the following:
 
+* You can run the new `+dfx identity get-principal+` command to return the principal associated with the current identity.
++
+Note that you must be able to provide a principal identifier for either the `+default+` or a specific identity to access the external {IC} network.
```

Comment:
This isn't necessarily true if you're a user, correct? You can still access a URL without providing the principal ID. As a developer, you are required to provide it in order to be provisioned a wallet canister and cycles to run your applications on the external IC network.

Reply:
I'm not really sure and can't test it because at the moment there's only access to the tungsten URL and you must provide credentials, but I can add the developer caveat.